airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -83,43 +86,87 @@ def client(self):
         )
         return session.client(service_name="secretsmanager", **self.kwargs)
 
-    def get_conn_uri(self, conn_id: str) -> Optional[str]:
+    def get_conn_uri(self, conn_id: str):
         """
         Get Connection Value
 
         :param conn_id: connection id
         :type conn_id: str
         """
-        return self._get_secret(self.connections_prefix, conn_id)
+        if self.connections_prefix and self.sep:
+            conn_id = self.build_path(self.connections_prefix, conn_id, self.sep)
 
-    def get_variable(self, key: str) -> Optional[str]:
+        try:
+            secret_string = self._get_secret(conn_id)
+            secret = ast.literal_eval(secret_string)
+        except ValueError:  # 'malformed node or string: ' error, for empty conns
+            connection = None
+
+        # These lines will check if we have with some denomination stored an username, password and host
+        if secret:
+            for user_denomination in ['user', 'username', 'login']:
```

Comment:
Can you extract this to a new method?

Reply:
yes avoid the nested blocks if possible, if it is unavoidable then just add `# pylint: disable=too-many-nested-blocks` at the end of the line where it complains.